randovania/games/prime1/hint_distributor.py

```diff
@@ -0,0 +1,24 @@
+from random import Random
+
+from randovania.game_description.game_patches import GamePatches
+from randovania.game_description.hint import (
+    HintLocationPrecision, HintItemPrecision,
+    PrecisionPair
+)
+from randovania.generator.filler.player_state import PlayerState
+from randovania.generator.filler.runner import PlayerPool
+from randovania.generator.hint_distributor import HintDistributor
+
+
+class PrimeHintDistributor(HintDistributor):
```

Comment:
Was this class needed right now?

Reply:
No, but I will most likely use it in my next PR. Would you like me to remove it?

Reply:
I was testing around with making other hints (this class works fine), but wanted to get this one hint out in time for the next release.

Reply:
If you plan to use then it's fine